website/content/en/docs/getting-started/users.md

```diff
@@ -28,40 +28,34 @@ Before you start, you must have access to a Kubernetes cluster. For example, you
 
 To set up ClusterLink on a Kubernetes cluster, follow these steps:
 
-1. {{< anchor create-fabric-ca >}}Create the Fabric's CA certificate and private key.
-
-    The ClusterLink Fabric is defined as all Kubernetes clusters (peers) that install ClusterLink gateways and can share services between the clusters, enabling communication among those services.
-    First, create the fabric Certificate Authority (CA):
+1. {{< anchor create-fabric-ca >}}Create the fabric's CA certificate and private key:
 
     ```sh
     clusterlink create fabric --name <fabric_name>
     ```
 
-    This command will create the CA files `cert.pem` and `key.pem` in a folder named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used.
-
-1. {{< anchor create-peer-certs >}}Create peer certificates.
+    The ClusterLink fabric is defined as all K8s clusters (peers) that install ClusterLink gateways and can share services between the clusters, enabling communication among those services.
+    This command will create the CA files `cert.pem` and `key.pem` in a directory named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used.
```

Comment:
I think the default was changed and is not `default_fabric` - please confirm with @kfirtoledo